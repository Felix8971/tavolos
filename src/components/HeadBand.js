import React from 'react';

class HeadBand extends React.Component {
  constructor(props) {
    super();    
  }
  

  render() {
    return (
      <div className='head-band-group'>
        <img className='head-band' src={'../images/Nox_Fish_jpg_1455851858.jpg'} />
        <div className='title-group'>
          <h1>Dine in the dark</h1>
          <h2>For the adventurous, looking to challenge your senses.</h2>
        </div>
      </div>
    )
  }
}

export default HeadBand;


