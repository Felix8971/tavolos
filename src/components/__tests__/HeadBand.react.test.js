import React from 'react';
import { shallow, mount, render } from 'enzyme';
import HeadBand from '../HeadBand';
import * as enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
enzyme.configure({ adapter: new Adapter() });

describe('HeadBand', () => {

    it('should be defined', () => {
      expect(HeadBand).toBeDefined();
    });

    it('should render correctly', () => {
      const tree = shallow(
        <HeadBand/>
      );
      expect(tree).toMatchSnapshot();
    });

});