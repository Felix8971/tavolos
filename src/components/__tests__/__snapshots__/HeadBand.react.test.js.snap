// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeadBand should render correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <HeadBand />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <img
          alt="a fish dish"
          className="head-band"
          src="./images/nox_fish.jpg"
        />,
        <div
          className="title-group"
        >
          <h1>
            Dine in the dark
          </h1>
          <h2>
            For the adventurous, looking to challenge your senses.
          </h2>
        </div>,
      ],
      "className": "head-band-group",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "alt": "a fish dish",
          "className": "head-band",
          "src": "./images/nox_fish.jpg",
        },
        "ref": null,
        "rendered": null,
        "type": "img",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <h1>
              Dine in the dark
            </h1>,
            <h2>
              For the adventurous, looking to challenge your senses.
            </h2>,
          ],
          "className": "title-group",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Dine in the dark",
            },
            "ref": null,
            "rendered": "Dine in the dark",
            "type": "h1",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "For the adventurous, looking to challenge your senses.",
            },
            "ref": null,
            "rendered": "For the adventurous, looking to challenge your senses.",
            "type": "h2",
          },
        ],
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <img
            alt="a fish dish"
            className="head-band"
            src="./images/nox_fish.jpg"
          />,
          <div
            className="title-group"
          >
            <h1>
              Dine in the dark
            </h1>
            <h2>
              For the adventurous, looking to challenge your senses.
            </h2>
          </div>,
        ],
        "className": "head-band-group",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "alt": "a fish dish",
            "className": "head-band",
            "src": "./images/nox_fish.jpg",
          },
          "ref": null,
          "rendered": null,
          "type": "img",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <h1>
                Dine in the dark
              </h1>,
              <h2>
                For the adventurous, looking to challenge your senses.
              </h2>,
            ],
            "className": "title-group",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Dine in the dark",
              },
              "ref": null,
              "rendered": "Dine in the dark",
              "type": "h1",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "For the adventurous, looking to challenge your senses.",
              },
              "ref": null,
              "rendered": "For the adventurous, looking to challenge your senses.",
              "type": "h2",
            },
          ],
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
